app/src/db/migrations/20241216000000_017-ches-logging.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable max-len */
+import stamps from '../stamps';
+
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return Promise.resolve().then(() =>
+    // Create public schema tables
+    knex.schema
+      .createTable('email_log', (table) => {
+        table.uuid('email_id').primary();
```

Comment:
PK's should share the name of the table. `email_log_ig`

Reply:
UUID updated.